src/mpid/ch4/src/ch4r_callbacks.c

```diff
@@ -549,7 +549,7 @@ int MPIDIG_send_cts_target_msg_cb(int handler_id, void *am_hdr, void *data, MPI_
         mpi_errno =
             MPIDI_SHM_am_isend_reply(MPIDIG_REQUEST(sreq, req->sreq).context_id,
                                      MPIDIG_REQUEST(sreq, rank), MPIDIG_SEND_DATA,
-                                     &send_hdr, sizeof(send_hdr),
+                                     &send_hdr, (MPI_Aint) sizeof(send_hdr),
                                      MPIDIG_REQUEST(sreq, req->sreq).src_buf,
```

Comment:
As we are always casting return of `sizeof`, do you think it might benefit if we define a utility macro for it? For example:
```
#define MPIR_SIZEOF(var) (MPI_Aint) sizeof((var))
```

Reply:
I think it is nice to have. But I am also aware there is another desire to reduce the macro usage. I think this is a case with marginal benefit and someone may even be against hiding the cast (another evil thing) behind a macro.